reporter/src/main/kotlin/reporters/StaticHtmlReporter.kt

```diff
@@ -146,6 +146,11 @@ class StaticHtmlReporter : Reporter() {
           color: #573a08;
         }
 
+        .ort-report-table tr.ort-hint {
+          background: #fffaf3;
+          color: #573a08;
```

Comment:
I'd propose to take the chance and set better colors, e.g.:

```
background: #f7f5ff;
color: #1c0859;
```

Reply:
Done (in a new commit, though).